ambry-frontend/src/test/java/com.github.ambry.frontend/FrontendIntegrationTest.java

```diff
@@ -156,7 +158,7 @@ public void healtCheckRequestTest()
     Queue<HttpObject> responseParts = nettyClient.sendRequest(httpRequest, null, null).get();
     HttpResponse response = (HttpResponse) responseParts.poll();
     assertEquals("Unexpected response status", HttpResponseStatus.OK, response.getStatus());
-    ByteBuffer content = getContent(response, responseParts);
+    ByteBuffer content = getContent(responseParts, HttpHeaders.getContentLength(response));
```

Comment:
Those lines were removed from `getContent` because the content length might be less than blob size for range requests. I thought it was easier to just pass in the expected size.  For the health check test, I will just pass in the length of the "GOOD" response